src/sql/src/plan/statement/dml.rs

```diff
@@ -761,6 +763,65 @@ pub fn describe_copy(
     .with_is_copy())
 }
 
+fn plan_copy_to(
+    scx: &StatementContext,
+    from: CopyToFrom,
+    to: &Expr<Aug>,
+    format: CopyFormat,
+    options: CopyOptionExtracted,
+) -> Result<Plan, PlanError> {
+    let aws_connection_name = match options.aws_connection {
+        Some(conn) => conn,
+        None => sql_bail!("AWS CONNECTION is required for COPY ... TO <url>"),
```

Comment:
I have had mixed feedback in my previous PRs about using `sql_bail!`. I am not exactly sure when this is okay vs when a new structured type should be created. Currently, it seems much more readable to keep these errors as unstructured `sql_bail!`, so using these here.